### DISCUSSION TITLE
List Of Games Which Uses A Lot of Something

### DISCUSSION THREAD

**Discussion by RedDevilus** [PCSX2/pcsx2#4794] [Information]
We need a list of games for easy testing for current and future PRs, like if it affects performance, timing or accuracy but only for certain games instead of testing every single game constantly. (To be edited) (Weak specs doesn't count but it will apply to current emulation annoyances and what the game would do on physical console)

Games that abuse;

EE:
- [ ] 

VU0:
- [ ] 

VU1: 
- [ ] 

COP2:
- [ ] 

VU Cache fills up quickly:
- [ ] Medal of Honor: European Assault
- [ ] Champions league 06/07
- [ ] Killzone
- [ ] Sega Superstars Tennis (but only the Space Channel 5 map)

GS (VRAM leak, high usage):
- [ ] Driv3r
- [ ] Colin McRae

Delay Slots
	Triple Branch
- [ ] Narnia 2: Prince Caspian
- [ ] Transformers: The Game
- [ ] Metal Gear Solid 2
- [ ] Dropship United Peace Force

**Comment by F0bes** [comment]
Colin McRae rally 3 hammers the GS thread for me.

**Comment by weirdbeardgame** [comment]
Katamari's always been a heavy hammer on the VU's

**Comment by seta-san** [comment]
stolen runs out of vram in hardware mode. also runs slow in software mode.
